materials/knitr.md

```diff
@@ -133,31 +139,22 @@ head(time_series)
 ggplot(time_series, aes(x = year, y = count)) +
   geom_point() +
   geom_line() +
-  facet_wrap(~species_id) +
-  scale_x_continuous(breaks = pretty_breaks(n=2))
+  geom_smooth() +
+  facet_wrap(~species_id)
 ```
 
-## A simple model
+### Notebook
```

Comment:
The code format continues here, to the end of the page, but I can't figure out why.